distribution/docs/src/main/resources/_contents/_developing/developing-catalog-api-contents.adoc

```diff
@@ -0,0 +1,20 @@
+
+=== Catalog Development Fundamentals
+
+This section introduces the fundamentals of working with the Catalog API the OGC Filter for Queries.
```

Comment:
Should that be "...the Catalog API and the OGC Filter...`?
Also...is that the letter 'O' or the number '0'? I can't find another capital 'O' for comparison.